public/src/utils.js

```diff
@@ -22,6 +22,7 @@
 			return (diff[0] * 1e3) + (diff[1] / 1e6);
 		};
 	} else {
+		winston = console;
```

Comment:
utils.js is on clientside too, so no winston, but this should work for `.warn`, `.log`, `.info` :D

Reply:
Neat!